docs/FAQ.md

```diff
@@ -111,6 +111,71 @@ store arbitrary files in `<your-git-repo>/scratch/`.
 
 You can find more details on `gitignore` files [here][gitignore].
 
+### How can I avoid committing my local-only changes to tracked files?
+
+Suppose your repository tracks a file like `config.json`, and you make some
+changes to that file to work locally. Since Jujutsu automatically commits the
+working copy, there's no way to Jujutsu from committing changes to the file.
+But, you never want to push those changes to the remote repository. 
+
+One solution is to keep these changes in a separate commit branched from the
+trunk and _merge_ the private commit into your branch to use those changes.
```

Comment:
Nit: If we use a merge commit here, we need to discuss that the _private commit_ is the second parent of your work. If we just say, that this should just be a parent, I think the whole section could be simplified.

Reply:
Here's a suggestion: 
"As long as your private commit is a parent of your work, it's changes will be reflected in the current commit and the working copy. As long as you keep this commit up to date, you can add any local changes there. "

Reply:
I worry that might be too much information before the reader sees what we're talking about.

I split this paragraph into two simpler sentences that are hopefully easier to read. At the bottom of the demo, I added some bullet points talking about the first and second parents. What do you think? Does this help?

Reply:
> What do you think? Does this help?

It seems good enough that I no longer see a reason to contest it.